packages/main/src/Option.js

```diff
@@ -68,6 +68,16 @@ const metadata = {
 			type: String,
 		},
 
+		/**
+		 * Defines the <code>additionalText</code>, displayed at the end of the option element.
+		 * @type {string}
+		 * @public
+		 * @since 1.0.0-rc.16
```

Comment:
not sure if the number is correct

Reply:
1.3.0
as the latest is 1.2.4, and this is a feature it will be released in the next minor version, which is 1.3.0

Reply:
thnx!

Reply:
Otherwise, no other comments, the code seems fine to me, perhaps I would also show the feature within the Select.sample.html page either as separate example or to integrate it with some of the existing ones.